openadapt/alembic/versions/f9586c10a561_migrate_data_to_new_fks.py

```diff
@@ -22,6 +22,7 @@ def upgrade() -> None:
     for table in [
         "action_event",
         "window_event",
+        # "browser_event",
```

Comment:
Should this be uncommented?

Reply:
Done here: [6643bc0](https://github.com/OpenAdaptAI/OpenAdapt/pull/744/commits/6643bc05bcf92c1445bc40e94517ac9bb0239385)